control-plane/config/crd/bases/consul.hashicorp.com_servicedefaults.yaml

```diff
@@ -234,6 +234,13 @@ spec:
                           upstream proxy instances will be monitored for removal from
                           the load balancing pool.
                         properties:
+                          enforcing_consecutive_5xx:
+                            description: EnforcingConsecutive5xx is the % chance that
+                              a host will be actually ejected when an outlier status
+                              is detected through consecutive 5xx. This setting can
+                              be used to disable ejection or to ramp it up slowly.
+                            format: int32
+                            type: integer
```

Comment:
One more quick question, and it's more that I don't know the answer than anything, but we're defining the `enforcing_consecutive_5xx` property as `int32` here (and everywhere) but have  as a `uint32` in `servicedefaults_typos.go`: 
```
EnforcingConsecutive5xx *uint32 `json:"enforcing_consecutive_5xx,omitempty"`
```

I don't think we'll ever have an overflow(underflow) here but it would make sense to have them both be the same type?

Reply:
Good point, I believe those yaml files were generated automatically via `make ctrl-manifests`. I can change them to `uint32` manually but I'm concerned about next time we run the script.

Reply:
I did a little bit of research and it looks like we're already using this model everywhere else since `int32` is used by Kubernetes for CRDs everywhere. So no changes needed!

Reply:
great, thanks!